UnitTests/ScientistTests.cs

```diff
@@ -7,17 +8,19 @@
 using UnitTests;
 using Xunit;
 
+using System.Reactive.Linq;
+
 public class TheScientistClass
 {
     public class TheScienceMethod
     {
         [Fact]
-        public void RunsBothBranchesOfTheExperimentAndReportsSuccess()
+        public async void RunsBothBranchesOfTheExperimentAndReportsSuccess()
```

Comment:
Ha ha good catch. I don't think it should matter, since this is just a unit test.  I made it async as I needed to use await. Interesting to see how easy it is to accidentally let a async void slip in. I can see the value of that reflected unit test checking for them.

I will change to async Task when I get back to my dev PC.